p2p/options.go

```diff
@@ -22,9 +22,9 @@ type ServerParameters struct {
 	WriteDeadline time.Duration
 	// ReadDeadline sets the timeout for reading messages from the stream
 	ReadDeadline time.Duration
-	// RangeRequestTimeout defines a timeout after which the session will try to re-request headers
+	// RequestTimeout defines a timeout after which the session will try to re-request headers
 	// from another peer.
-	RangeRequestTimeout time.Duration
+	RequestTimeout time.Duration
```

Comment:
This config breaking and might required config reset in the node
Cc @renaynay

Reply:
@renaynay should I revert to `RangeRequestTimeout` or we're fine with it?

Reply:
It's fine, we can break. Thank you for marking